modules/source/mail/config/mail.xml

```diff
@@ -34,7 +38,7 @@
 			store-uri="${protocol}://${username:}:${password:}@${host}:${port}/${folder}"
```

Comment:
Also, shouldn't thesr properties only apply when the mode is imaps?  I thought perhaps a profile based off the protocol would be the mechanism to apply the props